app/services/invite_provider_user.rb

```diff
@@ -12,7 +12,10 @@ def initialize(provider_user:)
   def call!
     lookup_provider_user
     invite_user_to_dfe_sign_in
+  end
 
+  def notify
+    lookup_provider_user
```

Comment:
Not sure this needs to be here, I cant think of any circumstances where we would still have an email at this stage.